platform/ui/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon, Typography } from '@ohif/ui';
+
+const Dropdown = ({ children, showDropdownIcon, list }) => {
+  const [open, setOpen] = useState(false);
+  const element = useRef(null);
+
+  const DropdownItem = ({ title, icon, onClick, index }) => {
+    const itemsAmount = list.length;
+    const isLastItem = itemsAmount === index + 1;
+
+    return (
+      <div
+        key={title}
+        className={classnames(
+          'flex px-4 py-2 cursor-pointer items-center transition duration-300 hover:bg-secondary-main',
+          {
+            'border-b border-secondary-main': !isLastItem,
+          }
+        )}
+        onClick={() => {
+          setOpen(false);
+          onClick();
+        }}
+      >
+        {!!icon && <Icon name={icon} className="text-white w-4 mr-2" />}
+        <Typography>{title}</Typography>
+      </div>
+    );
+  };
+
+  DropdownItem.defaultProps = {
+    icon: '',
+    onClick: () => {},
+  };
+
+  DropdownItem.propTypes = {
+    title: PropTypes.string.isRequired,
+    icon: PropTypes.string,
+    onClick: PropTypes.func,
```

Comment:
Pointless without click? Should be required?

Reply:
yes, it actually should be required even to avoid all those defaultProps/destructuring assignment for onClick. Thanks for remembering 🤦  -- thanks